dali/python/backend_impl.cc

```diff
@@ -485,7 +490,9 @@ void ExposeTensorList(py::module &m) {
 
       This function can only be called if `is_dense_tensor` returns `True`.
       )code",
-      py::return_value_policy::reference_internal);
+      py::return_value_policy::reference_internal)
+    .def("as_dlpack", &TensorListToDLPackView<GPUBackend>,
```

Comment:
Add some docs with the limitations. Some example how to use it would be nice as well.
Also we may want to rework the FW iterators to utilize this instead of doing a copy (as DALI owns the returned FW tensors anyway).